js/runtime.js

```diff
@@ -572,16 +566,40 @@
         },
 
         shape: {
-            fillShape: function(shp){
-                shp();
+            fill: function(pathSet){
```

Comment:
So the fill shape block takes a shape (which is the same as a pathset, I just didnt change the variable name).
In  Canvas2D you draw a path with, say, ctx.lineTo(somePoint), but that path wont actually show up until you call ctx.fill(), or ctx.storke(). But I didnt want fill and stroke to be a separate step block, and I didnt want the user to be able to "draw" paths without them showing up on the canvas. So I made fill and stroke the steps, and the things that actually draw the shapes the expressions.
That meant that the fill and stroke blocks do have to contain an expression (which I am assuming means that their corresponding function has to contain an argument), but all the actual "drawing" work has been done already, we just have to tell the canvas to fill (or stroke) the paths being drawn